frontend/src/libs/components/button/styles.module.scss

```
@@ -0,0 +1,13 @@
+.btn {
```

```suggestion
.button {
```
Abbreviations are prohibited according to the rule B16

---

done